projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.html

```diff
@@ -96,14 +96,14 @@
         </ng-template>
         <ng-container *ngTemplateOutlet="hasPinnedRecords && isRowPinningToTop ? pinnedRecordsTemplate : null"></ng-container>
         <ng-template igxGridFor let-rowData [igxGridForOf]="data
-        | treeGridTransaction:id:pipeTrigger
+        | treeGridTransaction
```

Comment:
The ID and pipetriggers should be here, because they cause the pipe to retrigger after the trigger is incremented, or the id is changed. I'm not sure if the ID is necessary, but the pipetrigger definitely is, even though it's not used inside the pipe.

Reply:
The pipeTrigger is definitely necessary, not sure about the id, but restoring both.